src/Helpers/getDefinedFilters.ts

```diff
@@ -0,0 +1,13 @@
+// tslint:disable-next-line
+import '../Extensions/Array'
```

Comment:
absolute paths are not working?

Reply:
I wonder why this didn't get picked up by ts-lint...?

Reply:
The absolute path wasn't working for some reason. I added the line to ignore the rule, because it wasn't letting me build with the absolute path.

Reply:
I'll try to debug it